functions/transformDataModel.js

```diff
@@ -391,6 +391,12 @@ async function handleKeyResults(doc) {
     delete data.description;
   }
 
+  if (!data.auto) {
+    delete data.sheetId;
+    delete data.sheetName;
+    delete data.sheetCell;
+  }
```

Comment:
This slows down the whole transform-function so not sure if we should go for it. And sometimes the transform-function just fails....

Reply:
I'm thinking of reverting this